src/view/items/component/getComponentConstructor.js

```diff
@@ -12,8 +12,8 @@ export default function getComponentConstructor ( ractive, name ) {
 	if ( instance ) {
 		Component = instance.components[ name ];
 
-		// if not from Ractive.extend or a Promise, it's a function that shold return a constructor
-		if ( Component && !Component.isInstance && !Component.then ) {
+		// if not from Ractive.extend or a Promise, it's a function that should return a constructor
+		if ( Component && !Component.isInstance && !Component.then && typeof Component === 'function' ) {
```

Comment:
You can add `isFunction` to the import with `isString` to get `isFunction( Component )`, and you could actually just replace the truthy check with that.

Reply:
Ok, I got it.